Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -357,11 +357,15 @@ logOutPopUpModelConfig state =
           optionButtonOrientation = "VERTICAL"
           , backgroundClickable = true
           , customerTipAvailable = true
+          , fareEstimateText =  getString FARE_ESTIMATE
+          , tipSelectedText = getString TIP_SELECTED
```

Comment:
Avoid using multiple concatenate

Above written code can be modified as 
if state.data.rateCard.additionalFare == 0 then "₹" <> show state.data.suggestedAmount  else "₹" <> show state.data.suggestedAmount)<> " - ₹" <> (show  state.data.suggestedAmount + state.data.rateCard.additionalFare)

Reply:
done